### PR TITLE
fix: publish types to NPM

### PR DESCRIPTION
**Pull request by lottamus**
Fixes https://github.com/bcnmy/insta-exit-sdk/issues/10

**npm run build**

image

**npm publish**

image

**dist/index.d.ts**

image

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -310,7 +309,7 @@ class Hyphen {
             }
 
             const tokenContract = new ethers.Contract(tokenAddress, erc20ABI, provider);
-            const tokenContractInterface = new ethers.utils.Interface(erc20ABI);
+            const tokenContractInterface = new ethers.utils.Interface(JSON.stringify(erc20ABI));
```
Comment (lottamus) [inline src/index.ts:312]: We're also doing this over here: https://github.com/bcnmy/insta-exit-sdk/blob/1266fe60b7f5de32187e2d03b082da40421b63f1/src/meta-transaction/util.ts#L8

**Comment by lottamus** [comment]
@tomarsachin2271 any chance we can merge this one? 🙏

**Comment by tomarsachin2271** [comment]
Hey thanks for raising the PR, let me check this

**Comment by divyaN73** [comment]
Can you please increase the sdk version in package.json? Rest looks good. :)

**Comment by lottamus** [comment]
@tomarsachin2271 @divyaN73 I updated it to v0.0.21 in d98beb8c6291ea05d736251065959ac438bc9341

**Pull request merged**